rome_app/templates/rome_templates/essay_list.html

```diff
@@ -120,9 +127,35 @@
 
 {% block content %}
 <ul class="results">
+<h2>Essays</h2>
     {% for essay in essay_objs %}
-    <li class="row"><span><a href="{% url 'specific_essay' essay.slug %}">{{essay.title}}</a></span>
-    <br> {{essay.slug}} <br> </li>
+    {% if essay.is_note == False %}
+    <li value class="row"><span>{{essay.author}}, <a href="{% url 'specific_essay' essay.slug %}">{{essay.title}}</a></span>
+    <br> {{essay.slug}} <br> 
+    {{essay.preview}}
+     <div class = "related">
+      {% if essay.related_list %}
+      {% for work in essay.related_list %}
+       {% if not work.ppid %}
+    
+      {% endif %}
+      {% endfor %}
```

Comment:
Lines 138-142 don't seem to do anything - they can be removed if they're not necessary.